doc/user/data/sql_funcs.yml

```diff
@@ -837,14 +837,25 @@
     description: |
       Returns the name of the current database.
     unmaterializable: true
-  - signature: 'current_role() -> text'
+  - signature: 'current_catalog() -> text'
     description: |
-      Alias for `current_user`.
+      Alias for `current_database`.
     unmaterializable: true
+    version-added: v0.100
```

Comment:
`v0.100` has already been cut, and the next release will be [`v0.101`](https://github.com/MaterializeInc/materialize/blob/main/src/environmentd/Cargo.toml#L4) can you bump this?